config/sync/field.storage.node.layout_builder__layout.yml

```diff
@@ -1,4 +1,4 @@
-uuid: 833a0790-a592-43cb-90d6-f15509b9c74a
+uuid: ed8c3733-5cf1-4e6d-940e-350e7c8d76b8
```

Comment:
I'm not sure how much difference it makes, but if we make the uuids match wherever we can, it cuts down on the diffs between profiles which simplifies reconciling changes.  I'll defer to Mike on this sort of thing though.

Reply:
The uuid in this pr matches that of the production sites. At some point the UUID on soe_profile `8.x-1.x` changed for whatever reason. We have to keep UUIDs the the same especially for field storage. During configuration import, if the UUID is different, Drupal will delete the field storage and the database table and then recreate it. Doing that loose any client data.